rmgpy/pdep/network.py

```diff
@@ -75,32 +75,43 @@ class Network:
     
     """
     
-    def __init__(self, label='', isomers=None, reactants=None, products=None, pathReactions=None, bathGas=None):
+    def __init__(self, label='', isomers=None, reactants=None, products=None,
+                 pathReactions=None, bathGas=None, netReactions=None, T=0.0, P =0.0,
+                 Elist = None, Jlist = None, Ngrains = 0, NJ = 0, activeKRotor = True,
+                 activeJRotor = True, grainSize=0.0, grainCount = 0, E0 = None):
+        """
+        To initialize a Network object for running a pressure dependent job,
+        only label, isomers, reactants, products pathReactions and bathGas are useful,
+        since the other attributes will be created during the run.
+
+        The other attributes are used to reinstantiate the created network object
+        for debugging and testing.
+        """
         self.label = label
         self.isomers = isomers or []
         self.reactants = reactants or []
         self.products = products or []
         self.pathReactions = pathReactions or []
         self.bathGas = bathGas or {}
-        self.netReactions = []
+        self.netReactions = netReactions or []
```

Comment:
The issue with using `[]` as a default parameter in a method occurs when the list is modified... the default option is now a list with an element (in the python coding). It's recommended that default options are always immutable to prevent that.

Reply:
Gotcha, good call! I was reading more about this subject--I was unaware that python initializes function defaults at function initialization time rather than function call time. I will keep this in mind for the future.